### DISCUSSION TITLE
blog/some-things-would-not-be-done-on-blog

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#31] [Announcements]
# blog/some-things-would-not-be-done-on-blog

哲学片段与学术之路

https://pathos.page/blog/some-things-would-not-be-done-on-blog

**Comment by mtfront** [comment]
看这篇之前恰好看了一篇字体载入有点慢的博客，想到了刚才看到字一个个歪歪扭扭蹦出来的体验再看到这篇深有体会。

**Comment by littleyz** [comment]
更正一点，“霞鹜文楷”应该不能归为手写体。

**Reply by Pathsis** [reply]
感谢提醒。似乎应该与仿宋这类字体相似。

**Reply by imkero** [reply]
霞鹜文楷是在 [Klee](https://github.com/fontworks-fonts/Klee) 的基础上补充缺失的中文字符。Klee 的简介里面就自称 `a script font handwritten by pencil or pen`，所以可以认为是手写体？

> Klee is a script font handwritten by pencil or pen. Its quiet design has an elegant look that sets itself apart from traditional script and textbook fonts. Ideal for body text.
>

**Comment by ghost** [comment]
CSS的`font-family`就是来解决客户端与源代码字体不一致的，浏览器只会把在当前机器上可用的字体应用到当前正在访问的网站上，如果字体不可用，那么就会用浏览器默认的字体代替，但是如果它加载第三方字体又强制声明`!important`就会导致不尊重客户端字体的情况了。

“霞鹜文楷”的流行让我想到了香港，为了“文化气息”，香港街道名牌居然换字体，令人更难看得清，这只为满足极少数人的喜好，还好的是浏览器上还是能用类似ublockorigin这样的插件屏蔽remote fonts.

**Reply by Pathsis** [reply]
我记得有`-apple-system`和`BlinkMacSystemFont`这样的字体设置来着。

博客网页字体因为是用来文字阅读的，更不能随便用字体。在熟悉的字体下阅读，会更快更舒畅。相反，注意到字体并不是一件好事。

**Comment by imjeff** [comment]
十分赞同文中的观点。在使用字体上，没必要改变原生字体，在字号和行距上进行优化即可。更加没必要进行过多的修饰，以保证读者阅读体验为主。
去年也对类似的问题进行过探讨：https://yayu.net/3762.html

**Reply by Pathsis** [reply]
是的，原生字体是用户熟悉和习惯的字体，一旦改变，反而会影响阅读速度。

**Reply by imjeff** [reply]
是的，保留原有阅读习惯，把字体的定义交给用户自行决定。突然遇到不习惯的字体，会有点“阅读障碍”。

**Comment by soulizer** [comment]
有两种博客我不喜欢：一种是广告铺天盖地的（尤其是弹窗广告，以及博客的大部分内容都是软广告）；还有一种就是言之无物，啥都要公开发篇文章来记录的（纯粹的记录，没有任何观点或思考，比如：午饭吃了什么，今天去了哪里……）。

不过，那毕竟是人家的自由，我不喜欢，别去看就好了。

我也不喜欢那些花里胡哨的字体。其实不只是字体，博客的配色、布局、交互逻辑等，但凡是过于标新立异的，都会显得喧宾夺主，影响阅读体验。

但这些都是博客外观设计的重要元素，尤其是字体。毕竟，独立博客相比于其他许多平台的一大优势就是可以更加自由地展示自己想要展示的风格。更何况，现在有一些阅读博客的方式（比如RSS订阅或者使用浏览器的“阅读模式”可以屏蔽掉除文章内容以外的其他元素）仍然可以让读者以自己熟悉的方式去阅读这些内容，而不受博主在文字之外下的功夫的影响。

所以我认为，在博客中使用较为常规并且符合自己文字风格的字体并无不妥，比如宋体、楷体、黑体，而不是那种七扭八歪、头顶长草、冒泡、比例严重失调，以及许多字无法正常显示，以至于这部分文字只能以默认字体显示，导致整篇文章看起来非常割裂。

就我个人而言，让评论者留下自己的网址是有益无害的。

首先，我和其他访客如果觉得这条评论有意思，大概率会想看看这个评论者创作的其他内容，通过评论者留下的网址无疑是最便捷的。

不排除确实有人只是为了留下他的网址而发布一些言之无物的评论，但这些似乎并不会对我造成任何打扰，顶多也就是多收到一封评论通知邮件。而且，如果我认为这个网址出现得不合时宜，也可以将其去除。

**Reply by Pathsis** [reply]
现在想想，第三方评论系统太破坏网站主题UI了。这也是为什么每次我部署好了twikoo或artalk后，还是放弃了。

**Comment by koobai** [comment]
博客相对来说是属于彰显自己的东西，每个阶段都有每个阶段的喜好，自己开心最重要，想怎么来就怎么来吧。
我们最多就是遇到不喜欢的，就关闭离开，遇到不错的，就多呆呆，多逛逛，简单纯粹。

**Comment by zmk-ink** [comment]
> 字体虽然重要，但最好是无感的。当普通用户打开网站，让其注意到字体的变化，可能并不是一件好事。

这句话我给五颗星 😊

**Comment by luckylele666** [comment]
啥叫有回访网址的评论系统。还有一条很重要，一定不能不支持 RSS！

**Comment by seeyousuperman** [comment]
我晕，要不是看到地址栏里是你的域名，我差点误以为这篇文章是我写的。  

几乎和我是完全相同的偏好，连偏好的细节都一致，这是我之前的about页面写的：    

1. 不使用任何统计工具，我不需要知道访客何处来，何处去，何时来，何时去；  
2. 没有动效，不必为你的设备配置低而担心风扇狂转；  
3. 不加入任何博客聚合（早期加入过个站商店）；  
4. 不用带有网址栏的评论系统，无法为你增加互访和流量（但我喜欢在别人的博客看看评论点点链接）；  
5. 如无真情，不必评论，冗余的评论影响界面美感 ； 
6. 如无必要，不必订阅；  
7. 不加广告，影响美感； 
8. 不使用类手写字体，觉得会视觉疲劳。  

---

现在已经简化了。
